### PR TITLE
#162414125 Components and actions for rating articles

### PR DESCRIPTION
**Pull request by ibraheemkabir**
#What does this PR do?
- This PR sets up the components for rating of articles
- This PR sets up the redux actions for rating an article
- This PR also carries out necessary css styles for the semantic rating component

#Description of Task to be completed?
-Create action file and axios calls for rating articles
-style articles rating component

#Related PT stories
#162414125

#Screenshots
ratings-component

### CODE REVIEW

**Comment by codrex** [comment]
screenshot 2019-03-02 at 9 13 22 pm

Why do you have commits for landing page logic on this PR?

File: src/pages/ViewArticle/index.jsx [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+import '../../components/widgets/Articlewidgets/widget.scss';
 
 export default function ViewArticle(props) {
   const { title } = props.match.params;
   return (
     <Fragment>
       <ReadArticle selectedArticle={title} />
       <div className=" articleReactions">
-        <Widgets selectedArticleId={title} />
+        <Widgets disabled={false} name={false} bselectedArticleId={title} ratingIcon={false} />
```
Comment (codrex) [inline src/pages/ViewArticle/index.jsx]: what is 'bselectedArticleId'?

Comment (ibraheemkabir) [reply src/pages/ViewArticle/index.jsx]: syntax error will fix that...

Comment (ibraheemkabir) [reply src/pages/ViewArticle/index.jsx]: fixed

**Review by codrex — APPROVED** [review]

**Pull request merged**